.github/CONTRIBUTING.md

```diff
@@ -1,68 +1,32 @@
-# Contributing to the FTC SDK
+# Contributing to this app
 
-The following is a set of guidelines for contributing the FIRST FTC SDK.  The FTC Technology Team welcomes suggestions for improvements to core software, ideas for new features, requests for built-in support of new sensors, and well written bug reports.
+## First thing you need: learn Git
```

Comment:
_hahahahah_

Reply:
I didn't like the idea that the contributing guidelines were there every time I opened a PR, and we didn't make them, so I changed them.